lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -5375,6 +5398,9 @@ class PullbackEmitter final : public SILInstructionVisitor<PullbackEmitter> {
   NO_ADJOINT(StrongRetainUnowned)
   NO_ADJOINT(DestroyValue)
   NO_ADJOINT(DestroyAddr)
+
+  // Class-related instructions.
+  NO_ADJOINT(ClassMethod) // Active only because it takes a class operand.
```

Comment:
How can it be active? The result is a function, and function will only be made active when we need their context cotangents.

Reply:
Oops, that was a total blunder, thanks.
Removed in https://github.com/apple/swift/pull/26313/commits/7aa294508721bfcc2437a157ea46e2df6e1d9dd0.